installation/gcp/dm/vmclarity-config.yaml

```diff
@@ -0,0 +1,16 @@
+imports:
+- path: vmclarity.py
+
+resources:
+- name: vmclarity
+  type: vmclarity.py
+  properties:
+    region: us-central1
+    zone: us-central1-f
+    machineType: e2-standard-2
+    sshPublicKey: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCyqIoOL9CcSxcNK/8YrDvgO2mfzKNWidKk6sYBMTEIVHqz3e1L/5kKi8sIh5ivSD+OiLlnmxfdc39l+5Ov7w9gODn/f57a85t8cPCS6Vths6wumwf4j5mYmwOS9ujmu2ZPJMc7P0i0CiTj12jniCglRm5C0v7W6VudBoRxpxxrz+n4LQyLiUicC6Qp/E7lyJFw1VYDLLSeakeapM+5M3dus00FLlWOOp7VvcE86K8h/31/d5QmqaLJHRX3oN9kiSXGXwijOgv1QYHO01/XFqZGa58yVGaSzT98o8gJDE4Bs5D8gpsepYUh6i18L1VH575F0v1MVG7qn5MmHp9nTh0T sambetts@SAMBETTS-M-H09X
```

Comment:
I think it would be better to commit vmclarity-config.yaml as an example using a comment instead of a specified ssh pubkey.